<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tunnel</class>
 <widget class="QWidget" name="Tunnel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="icon">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>icon</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name">
     <property name="placeholderText">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="local_port">
     <property name="maximum">
      <number>65365</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>8443</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="remote_address">
     <property name="placeholderText">
      <string>10.10.10.10:443</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="proxy_host">
     <property name="placeholderText">
      <string>proxy-host</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="browser_open">
     <property name="placeholderText">
      <string>https://127.0.0.1:8443</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="action">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
